src/test/java/uk/gov/ons/ctp/integration/envoycuc/context/StepsContext.java

```diff
@@ -23,26 +24,48 @@
    * hour and goes towards a count To avoid this, I assign a unique value as a prefix to the UPRN,
    * TelNO or IP Address. On @PostConstruct, the unique value is given as Date mmss and is a prefix
    * Each time String getUniqueValueAsString() is called - this value is incremented to give a truly
-   * unique value in tests where we don't care about that particular value
+   * unique value in tests where we don't care about that particular value.
+   *
+   * <p>Note that the prefix must be limited to 3 digits and <= 255, if used for a valid IPv4
+   * address octet.
    *
    * <p>dayHour is used where we want values within a test to be constant for THAT CONTEXT -
    * throughout the whole test This is so that we gain a fixed value for the test, but on rerun a
    * new value is created based on DDDHH and will be unique to the live rate limiter and so that
-   * tests will run as intended and will not be queered by a constantly running rate limiter.
+   * tests will run as intended and will not be queried by a constantly running rate limiter.
```

Comment:
The word is queered. Means altered or changed

Reply:
ah, I've learned something new